helpers/dcs.py

```diff
@@ -74,19 +120,33 @@ def current_leader(self):
 
     @abc.abstractmethod
     def touch_member(self, connection_string, ttl=None):
-        """touch_member"""
+        """Update member key in DCS.
+        This method should create or update key with the name = '/members/' + `~self._name`
+        and value = connection_string in a given DCS.
+
+        :param connection_string: how this instance can be accessed by other instances
+        :param ttl: ttl for member key, optional parameter. If it is None `~self.member_ttl will be used`
+        :returns: `!True` on success otherwise `!False`
+        """
 
     @abc.abstractmethod
     def take_leader(self):
-        """take_leader"""
+        """This method should create leader key with value = `~self._name` and ttl=`~self.ttl`
+        Since it could be called only on initial cluster bootstrap it could create this key not atomically."""
```

Comment:
I don't fully understand this sentence, does it mean that this function is only used during bootstrapping? 

could --> should?